### PR TITLE
Use consistent jargon while describing API

### PR DESCRIPTION
**Pull request by krisis**
Also, changed minPartSize reference to 64MiB everywhere including docs.

### CODE REVIEW

File: docs/API.md [hunk 1]
```diff
@@ -579,28 +579,28 @@ __Parameters__
...
 
 
 |Param   |Type   |Description   |
 |:---|:---| :---|
-|`objInfo.LastModified`  | _time.Time_  |modified time stamp. |
-|`objInfo.ETag` | _string_ |etag of the object.|
-|`objInfo.ContentType` | _string_ |Content-Type of the object.|
-|`objInfo.Size` | _int64_ |size of the object.|
+|`objInfo.LastModified`  | _time.Time_  |Time when object was last modified |
+|`objInfo.ETag` | _string_ |Etag of the object|
```
Comment (harshavardhana) [inline docs/API.md]: MD5 checksum of the object

File: docs/API.md [hunk 2]
```diff
@@ -639,22 +639,22 @@ if err != nil {
...
+|`bucketName`  | _string_  |Name of the bucket  |
+|`objectsCh` | _chan string_  | Prefix of objects to be removed   |
 
 
 __Return Values__
 
 |Param   |Type   |Description   |
 |:---|:---| :---|
-|`errorCh` | _chan minio.RemoveObjectError  | read objects deletion errors  |
+|`errorCh` | _chan minio.RemoveObjectError  | Channel of errors observed during deletion.  |
```
Comment (harshavardhana) [inline docs/API.md]: Why not "Errors observed during deletion" ?.  Channel might not be necessary.

Comment (harshavardhana) [inline docs/API.md]: No need to have "." at the end since we are removing that anyways

File: docs/API.md [hunk 3]
```diff
@@ -708,10 +708,10 @@ __Parameters__
...
 |Param   |Type   |Description   |
 |:---|:---| :---|
-|`bucketName`  | _string_  |name of the bucket.   |
-|`objectName` | _string_  |name of the object.   |
-|`expiry` | _time.Duration_  |expiry in seconds.   |
-|`reqParams` | _url.Values_  |additional response header overrides supports _response-expires_, _response-content-type_, _response-cache-control_, _response-content-disposition_.  |
+|`bucketName`  | _string_  |Name of the bucket   |
+|`objectName` | _string_  |Name of the object   |
+|`expiry` | _time.Duration_  |Expiry of presigned URL in seconds   |
+|`reqParams` | _url.Values_  |Additional response header overrides supports _response-expires_, _response-content-type_, _response-cache-control_, _response-content-disposition_.  |
```
Comment (harshavardhana) [inline docs/API.md]: Might be missing a comma here "Additional response header overrides, supports ... "

File: docs/API.md [hunk 4]
```diff
@@ -1006,36 +1007,36 @@ bucketNotification := BucketNotification{}
...
-	fmt.Println("Cannot set the bucket notification: " + err)
+    fmt.Println("Cannot set the bucket notification: " + err)
 }
 ```
 
 <a name="RemoveAllBucketNotification"></a>
 ### RemoveAllBucketNotification(bucketName string) error
 
-Remove all configured bucket notifications on a bucket.
+Remove all configured notifications on a bucket.
```
Comment (harshavardhana) [inline docs/API.md]: "Remove all configured bucket notifications" is correct..  "notifications" what is sent by the server.

File: docs/API.md [hunk 5]
```diff
@@ -1006,36 +1007,36 @@ bucketNotification := BucketNotification{}
...
+|`err` | _error_  |Standard Error  |
 
 __Example__
 
 
 ```go
 err := c.RemoveAllBucketNotification(bucketName)
 if err != nil {
-	fmt.Println("Cannot remove bucket notifications.")
+    fmt.Println("Cannot remove bucket notifications.", err)
```
Comment (harshavardhana) [inline docs/API.md]: "Unable to remove bucket notifications."

File: docs/API.md [hunk 6]
```diff
@@ -77,10 +77,10 @@ __Parameters__
 
 |Param   |Type   |Description   |
 |:---|:---| :---|
-|`endpoint`   | _string_  |S3 object storage endpoint.   |
-| `accessKeyID`  |_string_   | Access key for the object storage endpoint.  |
-| `secretAccessKey`  | _string_  |Secret key for the object storage endpoint.   |
-|`ssl`   | _bool_  | Set this value to 'true' to enable secure (HTTPS) access.  |
+|`endpoint`   | _string_  |Object storage endpoint   |
```
Comment (harshavardhana) [inline docs/API.md]: "S3 compatible object storage endpoint"

File: docs/API.md [hunk 7]
```diff
@@ -77,10 +77,10 @@ __Parameters__
...
+|`accessKeyID`  |_string_   |Access key for the object storage |
+|`secretAccessKey`  | _string_  |Secret key for the object storage |
+|`ssl`   | _bool_  | If 'true' API requests will be over HTTPS, and HTTP otherwise  |
```
Comment (harshavardhana) [inline docs/API.md]: "If 'true' API requests will be secure (HTTPS), and insecure (HTTP) otherwise"

File: docs/API.md [hunk 8]
```diff
@@ -128,30 +128,30 @@ Lists all buckets.
 
 | Param  | Type  | Description  |
 |---|---|---|
-|`bucketList`  | _[]BucketInfo_  | Lists bucket in following format shown below: | 
+|`bucketList`  | _[]BucketInfo_  | Lists buckets |
```
Comment (harshavardhana) [inline docs/API.md]: "Lists buckets" means this is doing something. In-fact this is return value so it should be "List of all buckets"

File: docs/API.md [hunk 9]
```diff
@@ -163,15 +163,15 @@ __Parameters__
...
 
 
 __Return Values__
 
 |Param   |Type   |Description   |
 |:---|:---| :---|
-|`found`  | _bool_ | indicates whether bucket exists or not  |
-|`err` | _error_  | standard error  |
+|`found`  | _bool_ | Indicates whether bucket exists or not  |
+|`err` | _error_  | Standard error  |
```
Comment (harshavardhana) [inline docs/API.md]: "Standard Error" consistent with below changes.. or change the below changes to "Standard error"

File: docs/API.md [hunk 10]
```diff
@@ -269,15 +269,15 @@ for object := range objectCh {
...
 
 __Parameters__
 
 
 |Param   |Type   |Description   |
 |:---|:---| :---|
-|`bucketName`  | _string_  |name of the bucket.   |
-| `objectPrefix` |_string_   | the prefix of the objects that should be listed. |
+|`bucketName`  | _string_  |Name of the bucket |
+| `objectPrefix` |_string_   | Prefix of objects to listed |
```
Comment (harshavardhana) [inline docs/API.md]: "Prefix of objects to be listed"

File: docs/API.md [hunk 11]
```diff
@@ -456,16 +456,16 @@ __Parameters__
 
 |Param   |Type   |Description   |
 |:---|:---| :---|
-|`bucketName`  | _string_  |name of the bucket.   |
-|`objectName` | _string_  |name of the object.   |
-|`reader` | _io.Reader_  |Any golang object implementing io.Reader.   |
+|`bucketName`  | _string_  |Name of the bucket  |
+|`objectName` | _string_  |Name of the object   |
+|`reader` | _io.Reader_  |Any Go type that implements io.Reader interface |
```
Comment (harshavardhana) [inline docs/API.md]: "Any Go type that implements io.Reader" - interface is not needed.

File: docs/API.md [hunk 12]
```diff
@@ -537,24 +537,24 @@ if err != nil {
...
 
 |Param   |Type   |Description   |
 |:---|:---| :---|
-|`bucketName`  | _string_  |name of the bucket.   |
-|`objectName` | _string_  |name of the object.   |
-|`filePath` | _string_  |file path of the file to be uploaded. |
-|`contentType` | _string_  |content type of the object.  |
+|`bucketName`  | _string_  |Name of the bucket  |
+|`objectName` | _string_  |Name of the object |
+|`filePath` | _string_  |path to file to be uploaded |
```
Comment (harshavardhana) [inline docs/API.md]: "path" is not capitalized.

File: docs/API.md [hunk 13]
```diff
@@ -579,28 +579,28 @@ __Parameters__
...
 
 |Param   |Type   |Description   |
 |:---|:---| :---|
-|`objInfo.LastModified`  | _time.Time_  |modified time stamp. |
-|`objInfo.ETag` | _string_ |etag of the object.|
-|`objInfo.ContentType` | _string_ |Content-Type of the object.|
-|`objInfo.Size` | _int64_ |size of the object.|
+|`objInfo.LastModified`  | _time.Time_  |Time when object was last modified |
+|`objInfo.ETag` | _string_ |Etag of the object|
+|`objInfo.ContentType` | _string_ |Content-Type of the object|
```
Comment (harshavardhana) [inline docs/API.md]: "Content type of the object."  - we don't need to write the http.Header canonical name here. Just like how we don't say "Content-Length of the object" instead we say "Size of the object".

File: docs/API.md (hunk 2 above)
Comment (harshavardhana) [inline docs/API.md]: Actually Channel is fine, since we are using it everywhere.

File: docs/API.md (hunk 3 above)
Comment (krisis) [inline docs/API.md]: No, there is no need for a comma there.

File: docs/API.md (hunk 4 above)
Comment (krisis) [inline docs/API.md]: Done.

File: docs/API.md (hunk 11 above)
Comment (krisis) [inline docs/API.md]: Done.

File: docs/API.md (hunk 10 above)
Comment (krisis) [inline docs/API.md]: Done.

File: docs/API.md (hunk 9 above)
Comment (krisis) [inline docs/API.md]: Done.

File: docs/API.md (hunk 8 above)
Comment (krisis) [inline docs/API.md]: Done.

File: docs/API.md (hunk 7 above)
Comment (krisis) [inline docs/API.md]: Done.

**Pull request merged**